Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -123,5 +237,43 @@ function rotateTable(obj, table, axisString, value)
                 table.setRowAtIndex(iRow,rowVec_rotated)
             end
         end
+        function outputPath = generateOutputPath(obj,path, ext)
+            % Function to generate an output path from no, partial, or fully
+            % defined user path. 
+            
+            % Validate the output filename
+            if size(path,2) > 1
+                % More than 1 input to the method, only takes one (filepath)
+                error([ num2str(size(path,2)) ' inputs, expecting zero or one'])
+            end
```

Comment:
This warning might belong in `writeTRC()`, which takes zero or one argument. This function (`generateOutputPath()`) takes 2 arguments.

Reply:
I think you put this method in `generateOutputPath()` to avoid duplicating the code in `writeTRC()` and `writeMOT()`. Perhaps you can just update the comments, then. Or make `generateOutputPath()` a private method.